daemon/control.proto

```diff
@@ -77,6 +77,9 @@ message ControllerToDaemon {
 		//This is a debugging feature!
 		GET_LAST_LOG = 5;
 
+		//Copies logs from /data/logs in trustx-cml to /var/cml-updates in c0
+		RETRIEVE_LOGS = 6;
```

Comment:
we should discuss, if we want to reuse the disabled GET_LAST_LOG functionality for this and transfer the logs through the protobuf socket. Then the caller from c0 can decide were the files should be placed inside its rootfs.

Reply:
I have now changed the functionality so that all logs from /data/logs are sent as protobuf messages from daemon to controller and are stored in a directory specified by the caller from c0.